cert/spec.go

```diff
@@ -150,8 +156,8 @@ func newSpecFromPath(path string) (*Spec, error) {
 }
 
 // Load reads a spec from a JSON configuration file.
-func Load(path, remote string, before time.Duration) (*Spec, error) {
-	spec, err := newSpecFromPath(path)
+func Load(path, remote string, before time.Duration, defaultServiceManager string, strict bool) (*Spec, error) {
```

Comment:
Same here, what's the purpose of `defaultServiceManager`?

Reply:
That's a refactoring screwup on my part.  This is supposed to be wired all the way down so that it is used instead of dummy.  Fixing.

Reply:
roughly, the code checking the action should have this in front of it:
```
if spec.ServiceManagerName == "" {
  spec.ServiceManagerName = defaultServiceManager
}
```

Reply:
makes sense!